boot.php

```diff
@@ -134,6 +134,40 @@
  */
 define('MAX_LIKERS',    75);
 
+/**
+ * @name Notify
+ *
+ * Email notification options
+ * @{
+ */
+/** @deprecated since 2020.12, use Notify\Type::INTRO instead */
+define('NOTIFY_INTRO',        1);
```

Comment:
Can you please replace the values by references to the moved constants?

Reply:
yeah.. I was too quick .. done ...